src/OrbitGl/ClientFlags.cpp

```diff
@@ -35,3 +35,5 @@ ABSL_FLAG(bool, show_return_values, false, "Show return values on time slices");
 
 ABSL_FLAG(bool, enable_tracepoint_feature, false,
           "Enable the setting of the panel of kernel tracepoints");
+
+ABSL_FLAG(bool, enable_sourcecode_view, false, "Enable the experimental source code view");
```

Comment:
nit: Since everywhere else its `SourceCode`
```suggestion
ABSL_FLAG(bool, enable_source_code_view, false, "Enable the experimental source code view");
```

Reply:
Done.